### PR TITLE
LPS-165662 dynamic-data-mapping-api: Add new fields to submission FormLayoutPage

### PR DESCRIPTION
**Pull request by gustavolimav**
https://issues.liferay.com/browse/LPS-165661
https://issues.liferay.com/browse/LPS-165662

Dont forward.

Variables names need to be discussed.

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:stop

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:stop

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:stop

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/model/DDMFormInstanceSettings.java [hunk 1]
```diff
@@ -125,6 +127,13 @@ public interface DDMFormInstanceSettings {
 	)
 	public boolean autosaveEnabled();
 
+	@DDMFormField(
+		label = "%body",
+		predefinedValue = "%you-can-fill-out-this-form-only-once.-contact-the-owner-of-the-form-if-you-think-this-is-a-mistake",
+		properties = "displayStyle=multiline", type = "localizable_text"
+	)
+	public String bodyText();
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/model/DDMFormInstanceSettings.java:135]: I think we could rename to "body". Do you agree?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi @gustavolimav, 
I finished my review. Let me know if you have any questions :smile:

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormPagesTemplateContextFactory.java [hunk 2]
```diff
@@ -177,8 +177,8 @@ private List<Object> _createColumnsTemplateContext(
 				List<String> ddmFormFieldNames =
 					ddmFormLayoutColumn.getDDMFormFieldNames();
 
-				ddmFormFieldNames.remove("headerText");
-				ddmFormFieldNames.remove("bodyText");
+				ddmFormFieldNames.remove("header");
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormPagesTemplateContextFactory.java:180]: Could you sort, please?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hey @gustavolimav, 
I finished your review. Please, let me know if you have questions :smile:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormPagesTemplateContextFactory.java (hunk 2 above)
Comment (gustavolimav) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormPagesTemplateContextFactory.java]: sure

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
Resent https://github.com/liferay-objects/liferay-portal/pull/1231 due CI failures that don't make sense

**Pull request closed without merging**